app/boards/shields/gherkin/Kconfig.defconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+if SHIELD_GHERKIN
+
+config ZMK_KEYBOARD_NAME
+    default "GHERKIN"
```

Comment:
Nitpick, is "GHERKIN" all caps the actual name? I've always seen it as "Gherkin".

Reply:
It's a fair cop, renamed to Gherkin